package.json

```diff
@@ -18,24 +24,18 @@
     "mem",
     "memory"
   ],
-  "author": "Richard Rodger (http://richardrodger.com)",
-  "contributors": [
-    "Richard Rodger (http://richardrodger.com)",
-    "Nicolas Herment"
-  ],
-  "license": "MIT",
   "dependencies": {
-    "lodash": "2.4.2"
+    "lodash": "3.10.1"
   },
   "devDependencies": {
-    "docco": "0.6.3",
+    "eslint-config-standard": "^4.4.0",
```

Comment:
Oh, no, but I know who does! @geek How do we run standard with Lab or do we need to use ESLint? I know it's the same underneath